css-naming-convention.md

```diff
@@ -0,0 +1,195 @@
+# CSS Naming Convention
+
+
+This is Atom's naming convention for creating UI in Atom and Atom packages. It's close to [BEM/SUIT](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md), but slightly customized for Atom's use case.
+
+
+## Example
+
+Below the commit box as a possible example:
+
+```html
+<div class='gitub-CommitBox'>
+	<div class='gitub-CommitBox-editor'></div>
```

Comment:
Inconsistent indentation here, this whole inner block needs to lose 6 spaces 😉.

Reply:
It's a tab. It shows up as 8 spaces to you because you haven't [installed this](https://chrome.google.com/webstore/detail/tab-size-on-github/ofjbgncegkdemndciafljngjbdpfmbkn?utm_source=chrome-app-launcher-info-dialog), probably. :stuck_out_tongue_closed_eyes:

Reply:
You're missing the bigger picture that whatever it is, it's not consistent through the entire block.